DOCS-3456 Adds Synthetics Dashboards Documentation (#14374)
DataDog/documentation: content/en/synthetics/dashboards/browser_test.md

---
title: Synthetics Browser Test Performance Dashboard
kind: documentation
description: Learn about the out-of-the-box Synthetics browser test performance dashboard.
further_reading:
- link: '/synthetics/ci_results_explorer'
  tag: 'Documentation'
  text: 'Learn about the CI Results Explorer'
---

## Overview

> The [browser test performance dashboard][1] provides insights about your browser test runs, browser analysis, web performance, and events. It is separated in three sections:

Reviewer: ```suggestion
The [browser test performance dashboard][1] provides insights about your browser test runs, browser analysis, web performance, and events. It shows:
```
